.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,4 +5,4 @@ To help speed up the process of merging your code, check the following:
 
 - [ ] There exists an issue discussing the need for this PR
 - [ ] I added new tests for the issue I fixed or built
-- [ ] I used `make commit` instead of `git commit`
+- [ ] I used `npm run cm` instead of `git commit`
```

Comment:
How about `cm`=>`commit`? You can also keep a `cm` shortcut but I think it's good to keep the commands self-explanatory